src/blocks/ui/insights/personalization.js

```diff
@@ -47,7 +62,8 @@ const Personalization = ( {
 	return (
 		<>
 			<div className="altis-analytics-block-metrics">
-				<DateRange ranges={ [ 7, 30, 90 ] } value={ days } onSetRange={ setDays } />
+				<DateRange
+					location="insights" ranges={ ranges } value={ days } onSetRange={ setDays } />
```

Comment:
If your editor is et to have a max line limit it's probably better to put all the props on new lines rather than split it like this.

```suggestion
				<DateRange
					location="insights"
					ranges={ ranges }
					value={ days }
					onSetRange={ setDays }
				/>
```